packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -51,6 +54,52 @@ export function getSourceToPermalink(blogPosts: BlogPost[]): {
   );
 }
 
+type LastUpdateOptions = Pick<
+  PluginOptions,
+  'showLastUpdateAuthor' | 'showLastUpdateTime'
+>;
+
+async function readLastUpdateData(
```

Comment:
Afaik this is duplicated from the docs plugin.
That can probably be moved to docusaurus utils to avoid duplication.

Also maybe move the type FileChange here too?